### PR TITLE
create WorkArrangement page

### PR DESCRIPTION
**Pull request by OlaShabalina**
Completed page as per: <blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;uMT9o9s9&#x2F;301-dev-work-arrangement-page">DEV: Work Arrangement Page</a></blockquote>

Decided to continue as one page for this one based on the Question2 as we won't be using it anywhere else.
Notes:
* No changes to data-model.
* No changes to layout or pictures. 
* MinW="750px"  same as for other pages till the Layout change

### CODE REVIEW

File: pages/form/WorkArrangement.jsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            <LinkButton
+              disabled={!workMode}
+              href={"/form/DaysOfTheWeekSelection"}
+              width={"105px"}
+              topMargin="0"
+              H="55px"
+              justifySelf="right"
+              onClick={() => saveDataAndShowLog("Next button clicked")}
+            >
+              Save
```
Comment (hqtan) [inline pages/form/WorkArrangement.jsx:178]: I think we've reverted back to using 'Next' for `LinkButton` text;
see [figma comment here](https://www.figma.com/file/Y42AKuHDW46LiLDaTOs6sM?node-id=978:55230#162988506) 
```suggestion
              Next
```

File: pages/form/WorkArrangement.jsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        left={["2", "10"]}
+      >
+        <BackButton
+          href="/"
+          onClick={() => {
+            saveDataAndShowLog("Back button clicked");
+          }}
+        />
+      </Box>
+      <Q3Cloud />
```
Comment (hqtan) [inline pages/form/WorkArrangement.jsx:74]: we probably need to change this cloud image to match the latest figma designs.
Might be easier to tackle that once all survey pages have been updated.
Have added the task of updating cloud pics to the [TODO list](https://trello.com/c/Nuhvxjes).

**Review by hqtan — APPROVED** [review]
thumbsup

**Comment by hqtan** [comment]
[DEV: Work Arrangement Page](https://trello.com/c/uMT9o9s9/301-dev-work-arrangement-page)

File: pages/form/WorkArrangement.jsx (hunk 2 above)
Comment (OlaShabalina) [reply pages/form/WorkArrangement.jsx:74]: Let's update them once the pages are ready. Also the progress bar, it's now different.

**Pull request merged**